lighthouse-core/gather/driver.js

```diff
@@ -277,9 +277,17 @@ class Driver {
     return new Promise((resolve, reject) => {
       this.sendCommand('Page.getAppManifest')
         .then(response => {
-          // We're not reading `response.errors` however it may contain critical and noncritical
-          // errors from Blink's manifest parser:
-          //   https://chromedevtools.github.io/debugger-protocol-viewer/tot/Page/#type-AppManifestError
+          // handle errors
+          if (response.errors) {
+            response.errors.forEach(error => {
+              log.warn('driver', 'Manifest error', error.message);
```

Comment:
not sure if we want to warn on all of these (if it's not a critical warning the manifest is still functional, and we don't warn on something like all uncaught exceptions), but don't feel strongly :)